vagrant/provision.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+#### Guest provisioner for use with Vagrant
+
+## This script will install the prerequisites for running the Colorado
+## RLA system on Red Hat Enterprise Linux 7 (RHEL 7) or CentOS.
+
+set -e
+
+### Basics
+
+## Required for SELinux policies on /vagrant that allow files within it to be
+## served by Apache httpd.
+yum -y install policycoreutils-python
+
+### Java
+
+yum -y install java-1.8.0-openjdk
```

Comment:
I'm assuming this gives the most recent patch version (like `1.8.0.172`.) Is that right?

Reply:
Not the most recent OpenJDK release, but the most recent patch level that CentOS (RHEL) supports. So yes *or* no, depending on how you were thinking about "most recent."